src/main/java/core/basesyntax/operations/PurchaseOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+
+public class PurchaseOperation implements OperationHandler {
+    @Override
+    public void updateDB(String fruit, int quantity, Storage storage) {
+        if (storage.getStorage().get(fruit) - quantity < 0) {
+            throw new RuntimeException("Quantity value can't be less than zero");
```

Comment:
uninformative message. If I want to buy 10 bananas and you have only 5 how will I know it? How many attempts do I need to do to find the maximum amount that I could buy?